src/DIRAC/Core/DISET/private/Service.py

```diff
@@ -360,6 +360,16 @@ def handleConnection(self, clientTransport):
       self._monitor.setComponentExtraParam('queries', self._stats['connections'])
     # TODO: remove later
     if useThreadPoolExecutor:
+      nQueued = self._threadPool._work_queue.qsize()
+      if nQueued > self._cfg.getMaxWaitingPetitions():
+        gLogger.error(
```

Comment:
Wouldn't a WARN be enough?

```suggestion
        gLogger.warn(
```

Reply:
Yeah I think either is fine, changed.